generic/tests/pktgen_perf.py

```diff
@@ -109,7 +109,7 @@ def install_package(ver, session=None):
         output_cmd(cmd_install)
         output_cmd(cmd_clean)
 
-    check_cmd = "uname -r |grep el8"
+    check_cmd = 'uname -r |grep -E "el8|el9"'
```

Comment:
We should have better solution, otherwise we will need to modify this for every new release. e.g.`if int(re.findall('el(\d)', process.getoutput('uname -r'))[0]) > 7:`

Reply:
@I have updated, please help review. 

 (1/1) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (156.32 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-03-12T02.42-6cf719e/results.html